packages/@uppy/core/src/Uppy.js

```diff
@@ -384,7 +384,8 @@ class Uppy {
 
   validateRestrictions (file, files = this.getFiles()) {
     try {
-      this.#restricter.validate(file, files)
+      this.#restricter.validate(file)
+      this.#restricter.validateTotals(files, [file])
```

Comment:
I think `validate` should always validate all. If you want to break it up into other things, then only call those other pieces elsewhere

Reply:
ok have changed validate back now to validate all, and introduced two methods for specific validation